src/components/Sidenav/ViewerSidenav/ViewerSidenav.tsx

```diff
@@ -26,16 +26,6 @@ const viewerSidenavItems: NavItemType[] = [
     name: 'New',
     to: absoluteRoutes.viewer.new(),
   },
-  {
-    icon: <SvgNavVideos />,
-    name: 'Videos',
-    to: absoluteRoutes.viewer.videos(),
-  },
-  {
-    icon: <SvgNavChannels />,
```

Comment:
Should have mentioned earlier but let's keep that link inside the navigation for now even though it's missing in the designs

Reply:
fixed